src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -149,62 +134,59 @@ const TableHeader: React.FC<TableHeaderProps> = ({
     };
   }, [leaveSeat]);
 
-  const updateTableTitle = useCallback(
-    async (newTableTitle) => {
+  const [updateResponse, updateTables] = useAsyncFn(
+    async (values: EditTableForm) => await updateTableInfo(values),
+    []
+  );
```

Comment:
I believe you can extract the individual props from `updateResponse` here as well.

```suggestion
  const [{ error }, updateTables] = useAsyncFn(
    async (values: EditTableForm) => await updateTableInfo(values),
    []
  );
```

Reply:
Good point. I'll keep destructuring from now on. Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a